test/mocks/event/wrapped_dispatcher.cc

```diff
@@ -0,0 +1,96 @@
+#include "test/mocks/event/wrapped_dispatcher.h"
+
+namespace Envoy {
+namespace Event {
+
+const std::string& WrappedDispatcher::name() { return impl_.name(); }
```

Comment:
nit: Consider moving the implementation of trivial forwarding functions to the header file.

Reply:
They're all trivial, so removed the .cc file.